android/modules/ui/src/java/ti/modules/titanium/ui/widget/picker/TiUINativePicker.java

```diff
@@ -81,9 +81,11 @@ private void styleTextView(int position, TextView tv) {
 		        fontProperties[TiUIHelper.FONT_STYLE_POSITION]);
 		    }
 		    if (rowProxy.hasProperty(TiC.PROPERTY_COLOR)) {
-		        final int color = TiConvert.toColor(rowProxy.getProperties(), TiC.PROPERTY_COLOR);
-		        tv.setTextColor(color);
-		    }
+				final int color = TiConvert.toColor(rowProxy.getProperties(), TiC.PROPERTY_COLOR);
+				tv.setTextColor(color);
+			} else {
+				tv.setTextColor(Color.WHITE);
```

Comment:
Instead of defining the text color as white, it would be better to obtain the default color and set it back to that.

```Java
private static int defaultTextColor = -1;
...
public static class TiSpinnerAdapter<T> extends ArrayAdapter<T>
{
    ...
    private void styleTextView(int position, TextView tv) {
        ...
        if (rowProxy.hasProperty(TiC.PROPERTY_COLOR)) {
            final int color = TiConvert.toColor(rowProxy.getProperties(), TiC.PROPERTY_COLOR);
            // obtain default text color
            if (defaultTextColor == -1) {
                defaultTextColor = tv.getCurrentTextColor();
            }
            tv.setTextColor(color);
        } else {
            tv.setTextColor(defaultTextColor);
        }
    ...
...